robot:
  # Joint state controller publisher
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controllers
  left_steer_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_left_steer
    pid: {p: 2.0, i: 0.01, d: 2}

  right_steer_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_right_steer
    pid: {p: 2.0, i: 0.01, d: 2}

  #Wheel Controllers
  front_left_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint_chassis_front_left_wheel
    pid: {p: 1.5, i: 1.0, d: 0.0}
    
  front_right_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint_chassis_front_right_wheel
    pid: {p: 1.5, i: 1.0, d: 0.0}



  left_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint_chassis_left_wheel
    pid: {p: 1.5, i: 1.0, d: 0.0}

  right_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint_chassis_right_wheel
    pid: {p: 1.5, i: 1.0, d: 0.0}